actors/market/src/lib.rs

```diff
@@ -1025,6 +1000,74 @@ impl Actor {
             }
         }
     }
+
+    fn process_deal_updates(
+        rt: &impl Runtime,
+        params: ProcessDealsParams,
+    ) -> Result<ProcessDealsReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let curr_epoch = rt.curr_epoch();
+        let mut total_slashed = TokenAmount::zero();
+        let mut expired_deals: Vec<DealID> = Vec::new();
+        let mut terminated_deals: Vec<DealID> = Vec::new();
```

Comment:
Did you leave terminations here to handle the transient-only case where a deal was marked terminated prior to this code, but then is manually processed before cron gets to it? Please document this. 

The FIP doesn't record this in the return value, because after the transition period there will never be any terminations here.

Reply:
Simplifying this by aborting the transaction if any terminated deals are found